include/spirv-tools/libspirv.h

```diff
@@ -375,6 +379,21 @@ spv_context spvContextCreate(spv_target_env env);
 // Destroys the given context object.
 void spvContextDestroy(spv_context context);
 
+// Creates a Validator options object with default options.
+spv_validator_options spvValidatorOptionsCreate();
+
+// Destroys the given Validator options object.
+void spvValidatorOptionsDestroy(spv_validator_options options);
+
+// Records the maximum number of structure members that is considered valid in
+// the given Validator options object. <options> argument must be non-null.
+void spvValidatorOptionsSetMaxStructMembers(spv_validator_options options,
+                                            const char* limit);
+
+// Returns the maximum number of structure members that is considered valid in
+// the given Validator options object. <options> argument must be non-null.
```

Comment:
"must be non-null"... and not a dangling pointer. :-)
How about "must be a valid options object".  And then in the creation method you say "returns a valid options object. The object remains vaild until it is passed into spvValidatorOptionsDestroy".